indico/modules/rb/schemas.py

```diff
@@ -401,6 +401,11 @@ def _check_room_name_format_placeholders(self, room_name_format, **kwargs):
             # validated client-side, no i18n needed
             raise ValidationError('Missing placeholders: {}'.format(', '.join(missing)))
 
+        placeholders = get_format_placeholders(room_name_format)
+        for ph in placeholders:
+            if ph not in ['building', 'floor', 'number']:
+                raise ValidationError(_('Unknown placeholder "{ph}"').format(ph=ph))
```

Comment:
Untested, but this can probably be simplified like this:

```python
if invalid := placeholders - {'building', 'floor', 'number'}:
    raise ValidationError(_('Invalid placeholders: {}').format(', '.join(invalid)))
```